test/utils/cludformation.js

```diff
@@ -57,9 +59,50 @@ function listStacks(status) {
   return awsRequest('CloudFormation', 'listStacks', params);
 }
 
+async function doesStackWithNameAndStatusExists(name, status) {
+  try {
+    const describeStacksResponse = await awsRequest('CloudFormation', 'describeStacks', {
+      StackName: name,
+    });
+    if (describeStacksResponse.Stacks[0].StackStatus === status) {
+      return true;
+    }
+    return false;
+  } catch (e) {
+    return false;
```

Comment:
Let's `return false` only on error that states that stack doesn't exist, otherwise we should throw error (maybe there are some creds issue, and that should be exposed)